### PR TITLE
rename binary in autotools to glimpse-*

### PR DESCRIPTION
**Pull request by igalic**
modify the m4 build code such that it will produce binaries named `glimpse-*`

### CODE REVIEW

**Comment by TrechNex** [comment]
Unfortunately I wasn't able to build this. I've attached the output I generated for this.

[dump.txt](https://github.com/glimpse-editor/Glimpse/files/3471964/dump.txt)

**Comment by TrechNex** [comment]
Unfortunately this doesn't seem to build. I think reading through that `libtool` is trying to relink a GNU IMP library that doesn't exist for some reason.

[dump.txt](https://github.com/glimpse-editor/Glimpse/files/3472086/dump.txt)

File: app/Makefile.am [hunk 1]
```diff
@@ -244,25 +244,25 @@ endif
 install-exec-hook:
 if DEFAULT_BINARY
 	cd $(DESTDIR)$(bindir) \
-	&& rm -f gimp$(EXEEXT) \
-	&& $(LN_S) gimp-$(GIMP_APP_VERSION)$(EXEEXT) gimp$(EXEEXT)
+	&& rm -f $(GLIMPSE_APP_NAME)$(EXEEXT) \
+	&& $(LN_S) $(GLIMPSE_APP_NAME)-$(GLIMPSE_APP_VERSION)$(EXEEXT) $(GLIMPSE_APP_NAME)(EXEEXT)
```
Comment (bochecha) [inline app/Makefile.am]: Did you forget the `` sign here (last `(EXEEXT)`)

Comment (TrechNex) [reply app/Makefile.am]: Fixed this typo, but sadly doesn't fix the build. :man_facepalming:

**Comment by TrechNex** [comment]
Thank you to @bochecha for spotting the problem! I'm going to do a fresh "from scratch" build. If it's all good I'll approve & merge :+1:

**Pull request merged**